### PR TITLE
Fixdvname

### PR DESCRIPTION
**Pull request by vhalbert**

### CODE REVIEW

File: import_images.pl [hunk 1]
```diff
@@ -78,8 +78,8 @@ =head1 Transfering Docker images
       target => "postgres_exporter:v0.4.7"
     },
     {
-      source => "$config->{registry}/fuse7-tech-preview/data-virtualization-server:$config->{tag_komodo}",
-      target => "data-virtualization-server:v0.0.5"
+      source => "$config->{registry}/fuse7-tech-preview/data-virtualization-server-rhel7:$config->{tag_komodo}",
+      target => "data-virtualization-server-rhel7:v0.0.5"
```
Comment (heiko-braun) [inline import_images.pl]: @nicolaferraro Is this right?

Comment (vhalbert) [reply import_images.pl]: RCM has changed recently to require  the os (e.g, -rhel7) at the end of the name.   This corresponds to the docker setting:  [http://pkgs.devel.redhat.com/cgit/containers/data-virtualization-server-openshift/tree/Dockerfile?h=fuse-7.4-openshift-rhel-7]

Comment (heiko-braun) [reply import_images.pl]: I know. The part that I wonder about is the target name. As part of the import we take it from `source` and push it to `target` name I think. So the templates would need to reflect that I wanted either @nicolaferraro or @rareddy to confirm this.

Comment (rareddy) [reply import_images.pl]: I am not familiar with fuse-online-install side, so I am not sure

Comment (heiko-braun) [reply import_images.pl]: I have fixed it here: https://github.com/syndesisio/fuse-online-install/commit/fa5012a30213d508bc8e95446771ef308877c546

**Pull request merged**